indico_zodbimport/modules/event_timetable.py

```diff
@@ -506,6 +530,75 @@ def _migrate_contribution(self, old_contrib, friendly_id):
         contrib._last_friendly_subcontribution_id = len(contrib.subcontributions)
         return contrib
 
+    def _migrate_abstract(self, old_abstract):
+        description = getattr(old_abstract, '_fields', {}).get('content', '')
+        description = convert_to_unicode(getattr(description, 'value', description))  # str or AbstractFieldContent
+        old_contribution = getattr(old_abstract, '_contribution', None)
+        type_ = old_abstract._contribTypes[0]
+        try:
+            type_id = self.legacy_contribution_type_map[type_].id if type_ else None
+        except KeyError:
+            self.importer.print_warning(cformat('%{yellow!}Invalid contrib type {}')
+                                        .format(convert_to_unicode(type_._name)),
+                                        event_id=self.event.id)
+            type_id = None
+
+        accepted_type_id = None
+        accepted_track_id = None
+
+        if old_contribution:
+            assert old_contribution.__class__.__name__ == 'AcceptedContribution'
+            if old_abstract._currentStatus.__class__.__name__ == 'AbstractStatusAccepted':
+                old_contrib_type = old_abstract._currentStatus._contribType
+                if old_contrib_type:
+                    accepted_type_id = self.legacy_contribution_type_map[old_contrib_type].id
+                accepted_track = old_abstract._currentStatus._track
+                accepted_track_id = int(accepted_track.id) if accepted_track else None
+
+        abstract = Abstract(event_new=self.event, legacy_id=old_abstract._id,
+                            description=description, type_id=type_id, accepted_track_id=accepted_track_id,
+                            accepted_type_id=accepted_type_id)
+
+        if old_contribution:
+            abstract.contribution = self.legacy_contribution_map[old_contribution]
+
+        if not self.importer.quiet:
+            self.importer.print_info(cformat('%{cyan}Abstract%{reset} {}').format(abstract.legacy_id))
+        # contribution/abstract fields
+        abstract.field_values = list(self._migrate_abstract_field_values(old_abstract))
+        abstract.judgements = list(self._migrate_abstract_judgements(old_abstract))
+        return abstract
+
+    def _migrate_abstract_judgements(self, old_abstract):
+        if not hasattr(old_abstract, '_trackJudgementsHistorical'):
+            self.importer.print_warning(
+                cformat('%{blue!}Abstract {} {yellow}has no judgement history!%{reset}').format(old_abstract._id),
+                event_id=self.event.id)
+            return
+        for track_id, judgements in old_abstract._trackJudgementsHistorical.iteritems():
+            seen_judges = set()
+            for old_judgement in judgements:
+                judge = old_judgement._responsible.user if old_judgement._responsible else None
```

Comment:
this might trigger extra queries to get the user, but i guess it wouldn't be a big issue because there are not that many conferences with abstracts